### DISCUSSION TITLE
Terms to cover new federal standards on race/ethnicity data collection

### DISCUSSION THREAD

**Discussion by sebastianduesing** [ufbmi/OMRSE#289] [Ideas]
Congress very recently approved revisions to the federal standards for race/ethnicity data collection ([info here](https://www.federalregister.gov/documents/2024/03/29/2024-06469/revisions-to-ombs-statistical-policy-directive-no-15-standards-for-maintaining-collecting-and)). Is OMRSE interested in creating terms to represent the new standards?

I'm involved with several projects that currently use OMRSE for race/ethnicity terms (including OBI and [HIPC Immunespace](https://immunespace.org)), and we would be very happy to have OMRSE terms for the new standards. We're more than willing to contribute, test, or otherwise help out with the development of these new terms, if OMRSE is willing to host them.

Thanks for your time!

**Comment by hoganwr** [comment]
Yes, we are absolutely interested. I just became aware of the planned changes. We typically discuss these issues at the monthly OMRSE meeting (first Wed of month at 11a ET).  Or if you'd like a separate meeting to discuss please let us know.

**Comment by hoganwr** [comment]
We intend to carry out this work, but it is somewhat involved in terms of the technical engineering. So we are converting this issue to a discussion thread, and then we will spin off specific tasks in the engineering workflow off as issues subsequently. Also the modeling that goes with this is written up in a draft manuscript, and we are asking the first author of the manuscript permission to share out either the current draft or some figures/diagrams (of relevance) from it.
